src/ripple/legacy/0.27/book/Quality.h

```diff
@@ -0,0 +1,146 @@
+//------------------------------------------------------------------------------
+/*
+    This file is part of rippled: https://github.com/ripple/rippled
+    Copyright (c) 2012, 2013 Ripple Labs Inc.
+
+    Permission to use, copy, modify, and/or distribute this software for any
+    purpose  with  or without fee is hereby granted, provided that the above
+    copyright notice and this permission notice appear in all copies.
+
+    THE  SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+    WITH  REGARD  TO  THIS  SOFTWARE  INCLUDING  ALL  IMPLIED  WARRANTIES  OF
+    MERCHANTABILITY  AND  FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+    ANY  SPECIAL ,  DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+    WHATSOEVER  RESULTING  FROM  LOSS  OF USE, DATA OR PROFITS, WHETHER IN AN
+    ACTION  OF  CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+    OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+*/
+//==============================================================================
+
+#ifndef RIPPLE_LEGACY_CORE_QUALITY_H_INCLUDED
+#define RIPPLE_LEGACY_CORE_QUALITY_H_INCLUDED
+
+#include <ripple/legacy/0.27/book/Amount.h>
+#include <ripple/legacy/0.27/book/Amounts.h>
+
+#include <cstdint>
+#include <ostream>
+
+namespace ripple {
+namespace legacy {
+namespace core {
+
+// Ripple specific constant used for parsing qualities and other things
```

Comment:
Wait, is this code being moved from ripple/app/book/Quality.h?  Have I just reviewed a lot of old code??? :-(

Let me know what's going on, this seems an exact duplicate of that file, so why is github showing this to me as new code?  Stopping here.